README.md

```diff
@@ -24,6 +24,12 @@ $ source env/bin/activate
 
 Deactivate your virtual environment with the `deactivate` command. As long as the environment is not removed, you may start sessions simply with the command `source env/bin/activate`.
 
+**Note**: The `requirements.txt` file contains transitive dependencies in addition to direct dependencies. To reconstruct the requirements from scratch, use the following commands to install the direct dependencies used for development, packaging, and releasing.
+
+```
+(env) $ pip install pytest-cov requests nox sphinx, sphinx-rtd-theme twine wheel
```

Comment:
should be "pip install pytest-cov requests nox sphinx sphinx-rtd-theme twine wheel" remove the ","